app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%- if controller_name != 'sessions' %>
+    <%= link_to t(".sign_in"), new_session_path(resource_name) %><br />
+  <% end -%>
```

Comment:
Why this file has a mix of `-` at the beginning & `-` end?

Reply:
Not entirely sure as well, could've been from the generator 🤔 Will remove it 👍